karma.conf.js

```diff
@@ -66,7 +68,7 @@ module.exports = function (config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['jsdom'],
```

Comment:
I remove Chrome from here as FF is a good default and is quicker to spin up (one of the benefits of using jsdom).